src/WebView.php

```diff
@@ -138,6 +138,9 @@ public function head(): void
      */
     public function beginBody(): void
     {
+        if ($this->getViewFile() === null) {
+            throw new \LogicException('View file cannot be null.');
```

Comment:
What do you think about to create `ViewFileNotFoundException`?

Reply:
That's not the case of view file not found... that's something like  `\RuntimeException('beginBody() should be called in views only')`...